src/status_im/ui/screens/wallet/address_add_edit/utils.cljs

```diff
@@ -0,0 +1,14 @@
+(ns status-im.ui.screens.wallet.address-add-edit.utils)
+
+(defn random-emoji
```

Comment:
I believe that emojis should be created more generic since they will be used in a lot of other places as well. Don't we already have then in the project?

Reply:
Yes, this can be a generic function to get random emoji from the emoji data set. 

The data is clojurized in `emoji-data` def var under `status-im2.contexts.emoji-picker.data` ns.

We can pass it directly to the `rand-nth` macro to get the random emoji. The result would be a map that will have a key named `unicode` which will have the emoji.

Reply:
done